recipes/coq-jupyter/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "coq-jupyter" %}
+{% set version = "1.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/coq_jupyter-{{ version }}.tar.gz
+  sha256: c8bee708c15bfa6d80c1c34e04617b794ce2b2dde4478af7679fa21ed92b15f5
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
Can this be `noarch: python`?

Reply:
There's a post-link.sh script that activates the jupyter extension.

Reply:
Ahh ok